### ISSUE TITLE
Contract between booth and store - Booth

### ISSUE THREAD

**Issue by jmartinacu** [EGC-23-24/decide-part-lorca#135]
We will unify the contract between the data exchanged by booth and store.
The contract between booth and store will be the object:

``` json
{
	voting: int,
	voter: int,
	vote: [
		{
			a: int,
			b: int,
		} 
	]
        voting_type: 'yesno'
}
```

**Comment by alvarobernal2412** [comment]
This structure should be modified in the yes/no voting and comment question voting

**Comment by alvarobernal2412** [comment]
This incident is solved in the PR #141

**Comment by alvarobernal2412** [comment]
This incident is solved in the PR #143 for the comment voting.
